### PR TITLE
append Traceback to the ticket created in report-issue of 500 page

### PR DESCRIPTION
**Pull request by sravfeyn**
For http://manage.dimagi.com/default.asp?133441

Implementation was trivial, but turning `DEBUG=False` on local setup was a mess. Following were two issues I spent most time to implement this (FYI for others)
- All HQ pages failed with `SuspiciousOperation: Invalid HTTP_HOST header` found that docs was not upto date and spent good amount of time on the issue before understanding the issue [here](http://getluky.net/2013/02/21/django-debugfalse-and-allowed_hosts/).
- After above fix, the pages are broken with no styling and no images, found that Django doesn't serve `static` when `DEBUG=False`, so I had to run the server with `--insecure` option

### CODE REVIEW

File: corehq/apps/hqwebapp/views.py [hunk 1]
```diff
@@ -152,10 +152,13 @@ def server_error(request, template_name='500.html'):
 
     # hat tip: http://www.arthurkoziel.com/2009/01/15/passing-mediaurl-djangos-500-error-view/
     t = loader.get_template(template_name)
+    import sys, traceback
```
Comment (dimagimon) [inline corehq/apps/hqwebapp/views.py]: E401 multiple imports on one line

File: corehq/apps/hqwebapp/views.py [hunk 2]
```diff
@@ -463,7 +467,10 @@ def bug_report(req):
         reply_to = settings.SERVER_EMAIL
 
     if req.POST.get('five-hundred-report'):
-        message = "%s \n\n This messge was reported from a 500 error page! Please fix this ASAP (as if you wouldn't anyway)..." % message
+        extra_message = "This messge was reported from a 500 error page! Please fix this ASAP (as if you wouldn't anyway)..."
```
Comment (dimagimon) [inline corehq/apps/hqwebapp/views.py]: E501 line too long (125 > 115 characters)

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by millerdev** [comment]
Test fail, kicked. Other than that this looks OK.

I'm a bit skeptical of passing the stack trace to the 500 page, but I guess as long as the exception value is not included it's only static information from our codebase and therefore shouldn't reveal any secrets.

**Comment by sravfeyn** [comment]
Yeah, thought we would store the exception somewhere for 5 mins and pass the key to the 500 page instead of the whole info, but went with this as most of the code is on github anyway. Not sure, what if the traceback passes through code in our private-repos. 

Also why is it a problem if Exception Values is exposed?

**Comment by millerdev** [comment]
Exception values often contain run-time information, which can include sensitive customer data.

**Pull request merged**